src/controllers/baseHandler.ts

```diff
@@ -0,0 +1,17 @@
+import { HELLO_COMMAND } from "../constants/commands";
+import { discordMessageRequest } from "../typeDefinitions/discordMessage.types";
+import { getCommandName } from "../utils/getCommandName";
+import { commandNotFound } from "./commandNotFound";
+import { helloCommand } from "./helloCommand";
+
+export function baseHandler(message: discordMessageRequest) {
+  const command = message.data.name.toLowerCase();
+  switch (command) {
+    case getCommandName(HELLO_COMMAND): {
```

Comment:
Are we expecting more commands to be added here? What if we will use if/else here?

Reply:
Yes, we are. The bot will have many more commands than just this one.